minio/mcs
# Configuration forms

## What does this do?
Implements forms (UI) for settings page:
- region
- cache
- compression
- etcd
- identity_openid
- identity_ldap
- policy_opa
- logger_webhook
- audit_webhook

Fixes a Memory leak in Configuration pages / Lambda Notification pages
Implements page for webhooks

NOTE: These forms are not connected to API yet

## How to test

- Go into configurations list, check the forms that work as expected without re-renders / hangs
- Simulate a send configuration & check that all the information in the form is in the headers
- Go into Lambda Notifications page & check that forms are working as expected
